### DISCUSSION TITLE
Reduce the explosive effects of oxygen/weldingfuel tanks

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#4943] [Ideas and suggestions]
1. Consider reducing the damage and structure damage caused by exploding welding tanks and o2 tanks.
Since they're now stackable it could potentially do a lot of harm, (accidental or caused by griefers).

2. A cook-off implementation for fuel/o2 tanks to prevent instant explosions if you ever so slightly got touched by fire would be great.

3. Additionally, tune the effects of tank explosions so they don't cause massive chain reactions (like your entire diving suit rack blowing up).

4. A different way to approach point 2 and 3 are to make tanks "fire/explosion proof" while in containers.

**Comment by Kulik242** [comment]
I don't think the tanks should be stackable in the first place. Based on their physical representation it is ridicules to carry 6 tanks in one slot.

**Reply by Xeavas** [reply]
I disagree with making tanks unstackable, especially toolbelts capacity was reduced from 12 to 6, but I felt that reducing the tanks stack size from 8 to 4-3 tanks in inventory (while stacking up to 8 in containers for storage) would make some sense as a compromise.

**Reply by Kulik242** [reply]
Stacking oxygen tanks trivializes the time management of diving missions. Consideration between more breathing time vs. more carrying capacity adds to the game ballance. Even the longest missions hardly require more than 3-4 reserve oxygen tanks. Making this a 1 slot requirement doesn't make sense gameplay vise nor do I like the idea of "magic pockets".
